npm/lit/src/mount.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  getContainerEl,
+  injectStylesBeforeElement,
+  setupHooks,
+  StyleOptions,
+} from '@cypress/mount-utils'
+import { html, LitElement, render, TemplateResult } from 'lit'
+import { unsafeHTML } from 'lit/directives/unsafe-html.js'
+
+export interface MountOptions<T extends HTMLElement>
+  extends Partial<StyleOptions> {
+  log?: boolean
+  properties?: Partial<T>
+}
+
+export type MountReturn<T extends keyof HTMLElementTagNameMap> =
```

Comment:
Users should use the `HTMLElementTagNameMap` then all available properties of the underlying object can be accessed after mounting.

```ts
declare global {
  interface HTMLElementTagNameMap {
    "counter-lit": LitCounter;
    "counter-wc": WebCounter;
  }
}
```